### ISSUE TITLE
Align numbers right

### ISSUE THREAD

**Issue by daniperez** [daniperez/gnome-mfa#1]
The titles are the same width in chars (padded with empty spaces when needed) but when they are translated into Gnome elements, they get different widths. 

![mfa](https://github.com/daniperez/gnome-mfa/raw/master/mfa.png)

**Comment by fadeouter** [comment]
just add `\\t` between name and number (better two tabs instead of one)

**Comment by daniperez** [comment]
yes, I already tried that. But tabs don't work as expected if the title is too wide. When I use a fixed size in characters, the on-screen width in Gnome is different than the fixed width I used. In the console the separation between label and number is correct but it changes when it is translated into Gnome means.

![tabs](https://cloud.githubusercontent.com/assets/579032/26173856/a532a688-3b4d-11e7-94d1-ede4c550244f.png)

**Comment by fadeouter** [comment]
easy way is to truncate long names with `cut -c -10` where 10 is how many symbols will remain
or use something like this: https://askubuntu.com/questions/523861/shorten-lines-adding-ellipsis

another way is to swap code and name

012345   Dropbox
234234   Some Great Service

**Comment by daniperez** [comment]
yep, that would work. The problem probably arises only if the string needs padding. If the string is shortened, it works in any case.

Thanks for the advice!
